woocommerce/woocommerce
# Remove Travis CI config, as WC migrated to GH actions.

### All Submissions:






### Changes proposed in this Pull Request:



Remove Travis CI config, as we've migrated to GH actions which are better integrated, faster and equally reliable.

TODO:
* [x] Update the coding standards check so that it actually detects problems.
* [ ] Update e2e package instructions for running on Travis to GH actions -> created a separate issue https://github.com/woocommerce/woocommerce/issues/29381
* [ ] Code coverage seems to depend on Travis, too. (postponed, to be worked on separately)

### How to test the changes in this Pull Request:

1. Check that Travis job no longer runs.
2. See the PHPCS actually runs https://github.com/woocommerce/woocommerce/pull/29233/files


### Other information:




### Changelog entry

> Removed Travis CI config.
